docs/BigQuery-connector.md

```diff
@@ -26,4 +26,6 @@ it can automatically be read from the environment. In other environments, the se
   auto-detect' when running on a Dataproc cluster. When running on other clusters, the file must be present on every
   node in the cluster.
 
-* **JSON**: Contents of the service account JSON file.
\ No newline at end of file
+* **JSON**: Contents of the service account JSON file.
+
+**Show Hidden Datasets:** Option to show hidden datasets.
```

Comment:
nit: Whether to show hidden datasets.

Also, is there a similar concept for BQ tables?

Reply:
No there is no such concept for table, any table starts with _ will show up in bigquery console